projects/sw-offline-google-analytics/src/lib/constants.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
     VERSION: 1
   },
   MAX_ANALYTICS_BATCH_SIZE: 20,
-  STOP_RETRYING_AFTER: 86400000, // One day, in milliseconds.
+  STOP_RETRYING_AFTER: 1000 * 60 * 60 * 48, // Two days, in milliseconds.
```

Comment:
It's a bit complicated to explain, but I'll give it a try.

Google Analytics processes all hits it receives at around 4 a.m. (the next morning) in the timezone of the view. The hit offset time specified via the `qt` param only works if processing for the target day has not occurred yet. Once hit processing is completed for a particular day, no further processing happens (except in the case of fixing errors due to bugs or other unusual circumstances).

This means, in general, the max `qt` time could be as short as 4 hours or as long as 28 hours (assuming normal processing schedules). I extended this to 48 hours just to be safe.

Alternatively we could just always send hits no matter what, and let Google Analytics process what it can.